charts/s1-agent/templates/cluster/cluster-role.yaml

```diff
@@ -14,4 +14,9 @@ rules:
 - apiGroups: ["batch", "apps"]
   resources: ["jobs", "deployments", "daemonsets", "replicasets", "statefulsets", "crontabs"]
   verbs: ["get", "watch", "list"]
+{{ if .Values.configuration.env.helper.agent_injection_enabled }}
```

Comment:
agent_injection_enabled is a long name that can be tiresome to set. can you change it to something like .Values.configuration.env.injection_enabled? even outside of the "helper" section

Reply:
done